src/secrets-kubernetes/src/lib.rs

```diff
@@ -7,37 +7,42 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-use anyhow::{bail, Error};
+use anyhow::{anyhow, bail, Error};
 use async_trait::async_trait;
-use k8s_openapi::api::core::v1::Secret;
-use k8s_openapi::apimachinery::pkg::apis::meta::v1::ObjectMeta;
+use k8s_openapi::api::core::v1::{Pod, Secret};
 use k8s_openapi::ByteString;
-use kube::api::{Patch, PatchParams, PostParams};
+use kube::api::{Patch, PatchParams};
 use kube::config::KubeConfigOptions;
-use kube::{Api, Client, Config};
+use kube::{Api, Client, Config, ResourceExt};
+use mz_ore::retry::Retry;
 use mz_secrets::{SecretOp, SecretsController};
+use rand::random;
 use std::collections::BTreeMap;
-use tracing::{error, info};
+use std::path::{Path, PathBuf};
+use std::time::Duration;
+use tracing::info;
 
 const FIELD_MANAGER: &str = "materialized";
-pub const SECRET_NAME: &str = "user-managed-secrets";
+const POD_ANNOTATION: &str = "secret-refresh";
```

Comment:
I think we should use the same namespacing that we do for our labels, so `materialized.materialize.cloud/secret-refresh`.

Reply:
+1 to namespacing this.
Is there a constant somewhere we can use for this?